src/index.html

```diff
@@ -5,8 +5,295 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="" class="header__link">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="header__logo"
+        >
+      </a>
+      <nav class="header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="" class="nav__link nav__link--is-active is-active">apple</a>
```

Comment:
Скорее всего можно так
```suggestion
            <a href="" class="nav__link is-active">apple</a>
```